apps/exports/tests/test_views.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+
+class StaffRequiredTestCase(TestCase):
```

Comment:
I appreciate you adding tests, but I'm not sure they are necessary if we do just use Django's built in decorators for this.

Reply:
Thanks. I agree.